### ISSUE TITLE
[Request] Please Provide Simple Example Mod

### ISSUE THREAD

**Issue by gudenau** [Kneelawk/GraphLib#54]
There are a lot of moving parts in this library and it would be nice to have a simple example to go along with it to show how everything works together.

**Comment by Kneelawk** [comment]
I am actually working on this right now. Along with the multi-module refactor for 1.21, I am working on adding some example mods that show how to use everything.

**Comment by Kneelawk** [comment]
For the time-being, I would recommend looking at [Extended Drawers](https://github.com/MattiDragon/ExtendedDrawers).

**Comment by gudenau** [comment]
I'm doing things similar to that and the graphs are ephemeral for me, any idea what's up with that?

**Comment by Kneelawk** [comment]
One thing is to do `/graphlib <universe> debugrender start` and see if the nodes are even getting added. Next, you probably want to make sure your BlockNodes implement proper `hashCode` and `equals` functions unless they're singletons.

**Comment by gudenau** [comment]
Yeah I turn that on, when I reload the save the graphs where gone.

I'll go double check that too.

**Comment by gudenau** [comment]
![image](https://github.com/Kneelawk/GraphLib/assets/5642329/7a40966d-ab75-421f-a05f-0f0c63785207)
![image](https://github.com/Kneelawk/GraphLib/assets/5642329/978f31cf-eeb2-4c57-8299-cce42c74b67b)
Yeah they are ephemeral.

**Comment by Kneelawk** [comment]
Are there any errors printed in the logs? Perhaps you forgot to add your block node types to your universe?

**Comment by gudenau** [comment]
Looks like that's the issue.

**Comment by Kneelawk** [comment]
https://github.com/Kneelawk/GraphLib/tree/wip/2.0-1/examples/multiblock-lamps now has some example code if you want to look at it.

**Comment by gudenau** [comment]
Okay I got them to persist.

There has to be a better place to discuss usage....
